src/asm3/person.py

```diff
@@ -511,7 +511,7 @@ def get_investigation(dbo, personid, sort = ASCENDING):
         sql += "ORDER BY o.Date DESC"
     return dbo.query(sql, [personid])
 
-def get_person_find_simple(dbo, query, username="", classfilter="all", includeStaff = False, includeVolunteers = False, limit = 0, siteid = 0):
+def get_person_find_simple(dbo, query, username="", classfilter="all", includeStaff = False, includeVolunteers = False, limit = 0, siteid = 0, datatype="all"):
```

Comment:
nothing really wrong with the implementation, but "datatype" is a bit of a confusing name. I would call it "typefilter"

Reply:
would also rearrange the parameters so that it's next to classfilter in the list for readability, then do a quick search for other places that call get_person_find_simple to make sure they're unaffected